Documentation/gettingstarted/hubble-enable.rst

```diff
@@ -51,6 +51,22 @@ networking infrastructure in a completely transparent manner.
               --set global.hubble.relay.enabled=true \\
               --set global.hubble.ui.enabled=true
 
+        .. note::
+
+           For some environments, if you want to expose Hubble-UI to your protected network - 
+           we may recommend you to use reverse proxy with TLS and OAuth enabled. 
```

Comment:
nit: can you add OAuth and UI into `spelling_wordlist.txt` by running `Documentation/update-spelling_wordlist.sh  OAuth UI` command.